transformers/test_data/vow_file.go

```diff
@@ -48,13 +48,13 @@ var rawVowFileLog = types.Log{
 var VowFileEventLog = core.EventLog{
```

Comment:
Super minor but maybe worth renaming this to specify `AuctionAttributes` as well